src/dbnode/storage/fs_merge_with_mem.go

```diff
@@ -117,6 +117,9 @@ func (m *fsMergeWithMem) ForEachRemaining(
 
 	for seriesElement := seriesList.Front(); seriesElement != nil; seriesElement = seriesElement.Next() {
 		seriesID := seriesElement.Value
+
+		// NB(r): Why isn't the tags just put in the linked list
```

Comment:
Maybe replace this with a `// TODO: investigate why the tag isn't put into the linked list element`?

Reply:
I think this just never got done and this was fast way to avoid doing it like I questioned why it's not being done like that.

Reply:
But yeah I'll change it to "we should blah"